source/numerics.tex

```diff
@@ -16034,3 +16034,2479 @@
                                      A, t, d, B, divides<void>{});
 \end{codeblock}
 \end{itemdescr}
+
+\rSec1[simd]{Data-parallel types}
+\rSec2[simd.general]{General}
+
+\newcommand\foralli[1][]{for all $i$ in the range of \range{0}{#1size()}}
+\newcommand\Foralli[1][]{For all $i$ in the range of \range{0}{#1size()}}
+\newcommand\forallmaskedi{for all selected indices $i$ of \tcode{mask}}
+
+\newcommand\ConstraintUnaryOperatorWellFormed[2][const ]{%
+  \constraints \tcode{requires (#1value_type a) \{ #2; \}} is \tcode{true}.
+}
+
+\newcommand\ConstraintOperatorTWellFormed{%
+  \constraints \tcode{requires (value_type a, value_type b) \{ a \placeholder{op} b; \}} is \tcode{true}.
+}
+
+\pnum
+[simd] defines data-parallel types and operations on these types.
+\begin{note}
+The intent is to support acceleration through data-parallel execution resources
+where available, such as SIMD registers and instructions or execution units
+driven by a common instruction decoder.
+\end{note}
+
+\pnum
+The set of \defn{vectorizable types} comprises all standard integer types,
```

Comment:
Please use \defnadjx so that we get index entries for both "vectorizable type" and "type, vectorizable".